armi/reactor/tests/test_components.py

```diff
@@ -167,14 +168,66 @@ class TestComponent(TestGeneralComponents):
 
     componentCls = Component
 
-    def test_initializeComponent(self):
-        expectedName = "TestComponent"
-        actualName = self.component.getName()
+    def test_initializeComponentMaterial(self):
+        """Creating component with single material.
+
+        .. test:: Components are made of one material
+            :id: T_ARMI_COMP_1MAT0
+            :tests: R_ARMI_COMP_1MAT
+        """
         expectedMaterialName = "HT9"
         actualMaterialName = self.component.material.getName()
-        self.assertEqual(expectedName, actualName)
         self.assertEqual(expectedMaterialName, actualMaterialName)
 
+    def test_initializeComponentName(self):
+        """Creating component and retrieving the name.
+
+        .. test:: Composite name is accessible
+            :id: T_ARMI_COMP_1MAT1
+            :tests: R_ARMI_COMP_1MAT
+        """
+        expectedName = "TestComponent"
+        actualName = self.component.getName()
+        self.assertEqual(expectedName, actualName)
```

Comment:
Delete this test.

The requirement being referenced is not related to the test.